app/stores/toplevel/LoadingStore.js

```diff
@@ -134,7 +134,9 @@ export default class LoadingStore extends Store {
         address => {
           // TODO: validation should be done based on wallet type
           const addressKind = tryAddressToKind(address, 'bech32', networks.CardanoMainnet);
-          const valid = addressKind != null && addressKind === CoreAddressTypes.CARDANO_LEGACY;
+          const valid = addressKind != null &&
+          ( addressKind === CoreAddressTypes.CARDANO_LEGACY ||
+            addressKind === CoreAddressTypes.CARDANO_BASE);
```

Comment:
Instead of only hard-coding base addresses, it probably makes more sense to use `isValidReceiveAddress`